src/Components/MyBooks/MyBooks.tsx

```diff
@@ -55,7 +65,7 @@ const MyBooks = ( { currentUser }: { currentUser: User | any}) => {
   const [ availLibrary, setAvailLibrary ] = useState([])
   const [ unavailLibrary, setUnavailLibrary ] = useState([])
   const [ pendingRequests, setPendingRequests ] = useState([])
-
+  const [ deleteBook ] = useMutation(DELETE_BOOK)
   
   useEffect(() => {
```

Comment:
Ya know... I just saw this and it might be from someone else (my) branch... but do you know whats going on here?